tetris.js

```diff
@@ -229,49 +242,63 @@ module.exports = {
 		return newArr;
 	},
 	
-	buildTetromino: function(stringOfCoordinates){
-		let math = stringOfCoordinates.split("\n")
-		let arrayOfCoordinates = []
+	getCoordinates: function(stringOfCoordinates){
+		let fieldArray
 		let quantityElem = 0;
+		let arrayOfCoordinates = []
+		if(typeof(stringOfCoordinates) == "string"){
+			fieldArray = stringOfCoordinates.split("\n")
+		}else{
+			fieldArray = stringOfCoordinates
+		}
 		
-		for(let i = 0, elem_Y = 0; i < math.length; i++){
-			if(math[i]!= false){
+		for(let i = 0, axisElement_Y = 0; i < fieldArray.length; i++){
```

Comment:
this is not element, but index. Element is for elements of some list-data-structure, like array

Reply:
I did